superbench/common/utils/topo_aware.py

```diff
@@ -0,0 +1,211 @@
+
+import re
+import networkx as nx
+
+
+class quick_regexp(object):
+    """
+    Quick regular expression class
+
+    """
+
+    def __init__(self):
+        self.groups = None
+        self.matched = False
+
+    def search(self, pattern, string, flags=0):
+        match = re.search(pattern, string, flags)
+        if match:
+            self.matched = True
+            if match.groups():
+                self.groups = re.search(pattern, string, flags).groups()
+            else:
+                self.groups = True
+        else:
+            self.matched = False
+            self.groups = None
+
+        return self.matched
+
+
+def gen_topo_aware_config(hosts, ibstat_file, topology_file, config_path):
```

Comment:
Do you use `config_path` in this function?

Reply:
Thanks for catching it. Fixed.